Description: Adjusts the bundle plugin configuration to avoid generating an empty jar
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/pom.xml
+++ b/pom.xml
@@ -70,7 +70,6 @@
            <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
            <Export-Package>org.codehaus.jettison*;version=${project.version}</Export-Package>
            <Import-Package>*</Import-Package>
-           <Private-Package>!*</Private-Package>
            <Implementation-Title>${project.name}</Implementation-Title>
            <Implementation-Version>${project.version}</Implementation-Version>
          </instructions>
